### ISSUE TITLE
Preserve flow execution after 20 minutes if it do any activity

### ISSUE THREAD

**Issue by kirill-levitskiy** [elasticio/sailor-nodejs#129]
There is an issue with a flow processing that stops accidentally after 20 minutes of execution.
Here is a line in the code that describes a constant for timeout in case of Running flow after 20 minutes
https://github.com/elasticio/sailor-nodejs/blob/95a667e9b5a254d2ae22e88a8c04b663ad4bd0c1/lib/sailor.js#L15

The case:
1. Create a flow;
2. First step generates an array with 7000 int values;
3. Second step (code-component) emits each array item individually with delay (500 ms).
4. Run a flow.

An execution will continue 20 minutes and it stop emit messages even if an amount is less than 7000.

As a solution, there should be checked any activity during last 20 minutes before firing a timeout operation in order to keep a flow alive if it is still doing any job.

**Comment by zubairov** [comment]
AFAIK this time-out is for a _single_ message, not for the whole execution. However let's see if @elasticio/platform-qa could reproduce this issue

**Comment by drobiazko** [comment]
@kirill-levitskiy I remember we had an event called `heartbeat` which is used to prolong the timeout based shutdown. Do we still have it?

**Comment by jhorbulyk** [comment]
Similar to https://github.com/elasticio/sailor-nodejs/issues/115

**Comment by pnedelko** [comment]
@drobiazko no, we haven't migrated `heartbeat` to the Platform v2 (Sailor).
